pkg/operation/botanist/controlplane/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -302,7 +302,7 @@ func (k *kubeControllerManager) Deploy(ctx context.Context) error {
 			ContainerPolicies: []autoscalingv1beta2.ContainerResourcePolicy{{
 				ContainerName: kubeControllerManagerContainerName,
 				MinAllowed: corev1.ResourceList{
-					corev1.ResourceCPU:    resource.MustParse("25m"),
+					corev1.ResourceCPU:    resource.MustParse("100m"),
```

Comment:
FYI: `25m` was used based on the suggestion in https://github.com/gardener/gardener/pull/3057#discussion_r513640869 (cc @amshuman-kr)
But I'm fine with increasing it again to stabilize the control planes.